common/src/main/res/drawable/ic_staking.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="32dp" android:viewportHeight="33"
+    android:viewportWidth="32" android:width="32dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#ffffff" android:fillType="evenOdd" android:pathData="M16,32.5C24.8366,32.5 32,25.3366 32,16.5C32,7.6634 24.8366,0.5 16,0.5C7.1634,0.5 0,7.6634 0,16.5C0,25.3366 7.1634,32.5 16,32.5ZM25.2077,11.2531L17.1566,6.7802C16.484,6.4066 15.516,6.4066 14.8434,6.7802L6.7923,11.2531C5.7359,11.84 5.7359,13.16 6.7923,13.7469L8.1479,14.5L6.7923,15.2531C5.7359,15.84 5.7359,17.16 6.7923,17.7469L8.1479,18.5L6.7923,19.2531C5.7359,19.84 5.7359,21.16 6.7923,21.7469L14.8434,26.2198C15.516,26.5934 16.484,26.5934 17.1566,26.2198L25.2077,21.7469C26.2641,21.16 26.2641,19.84 25.2077,19.2531L23.8521,18.5L25.2077,17.7469C26.2641,17.16 26.2641,15.84 25.2077,15.2531L23.8521,14.5L25.2077,13.7469C26.2641,13.16 26.2641,11.84 25.2077,11.2531ZM21.793,15.644L17.1566,18.2198C16.484,18.5934 15.516,18.5934 14.8434,18.2198L10.207,15.644L8.6661,16.5L10.207,17.356L12.2661,18.5L15.8147,20.4714C15.8832,20.5095 16.1168,20.5095 16.1853,20.4714L19.7339,18.5L21.793,17.356L23.3339,16.5L21.793,15.644ZM8.6661,20.5L10.207,19.644L14.8434,22.2198C15.516,22.5934 16.484,22.5934 17.1566,22.2198L21.793,19.644L23.3339,20.5L16.1853,24.4714C16.1168,24.5095 15.8832,24.5095 15.8147,24.4714L8.6661,20.5ZM16.1853,8.5286C16.1168,8.4905 15.8832,8.4905 15.8147,8.5286L8.6661,12.5L15.8147,16.4714C15.8832,16.5095 16.1168,16.5095 16.1853,16.4714L23.3339,12.5L16.1853,8.5286Z"/>
+</vector>
```

Comment:
This icon already exists (ic_substract)

Reply:
I checked all icons in common drawables. We don't have such icon in this module

Reply:
ic_substract is in the staking module, and ic_staking is in the wallet icon. Right now I can't access the icon from staking module. Do you think it is good idea to make those modules co-dependent in order to access icons?

Reply:
Yeah, it actually was a mistake to put icons into the staking module. It is better to put everything into common module to make it reusable and dont bother searching in different modules for collisions

Reply:
Done